### DISCUSSION TITLE
Traefik HTTPS settings

### DISCUSSION THREAD

**Discussion by DBLouis** [tarampampam/error-pages#299] [General]
The example in the README only shows simple HTTP configuration with Traefik. What is the proper way to configure Traefik and error-pages so that the error pages are used for both HTTP and HTTPS entrypoints? I have been trying to make it work for a while but in my setup, error pages are only shown for HTTP. For HTTPS I get the blank page with 404 when trying to access invalid subdomain.

**Comment by tarampampam** [comment]
For Traefik-specific configuration questions, the best place to get help is the [Traefik Community Forum](https://community.traefik.io/) and the [Traefik GitHub Issues](https://github.com/traefik/traefik/issues).

That said, the root cause is almost certainly that the `errors` middleware is attached only to the `web` (HTTP) entrypoint but not to `websecure` (HTTPS). Here are the links that should have everything you need:

- [Traefik: entrypoints configuration](https://doc.traefik.io/traefik/reference/install-configuration/entrypoints/) - see the `http.middlewares` option to attach a middleware globally to an entrypoint
- [Traefik: errors middleware reference](https://doc.traefik.io/traefik/reference/routing-configuration/http/middlewares/errorpages/)
